lib/peep.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg' 
+
+class Peep
+
+  attr_reader :id, :message
+
+  def initialize(id:, message:)
+    @id = id
+    @message = message
+  end
+
+  def self.all 
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps;")
+    result.map do |peep|
+    Peep.new(id: peep['id'], message: peep['message'])
+    end
+  end
+
+  def self.create(message:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
This piece of code appears twice in this file. How might you remove this duplication?